pkg/urfav_overrides/flags.go

```diff
@@ -0,0 +1,151 @@
+package cfflags
+
+import (
+	"errors"
+	"flag"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+)
+
+type Flags struct {
+	*flag.FlagSet
+	urFavFlags []cli.Flag
+}
+
+func New(name string, flags []cli.Flag, c *cli.Context) (*Flags, error) {
```

Comment:
If you could put a quick description as to why we need the override that would be fantastic - just to avoid causing confusion as to why this is here.